snprc_ehr/src/org/labkey/snprc_ehr/history/LabworkDataSource.java

```diff
@@ -190,12 +191,7 @@ protected String getPrimaryGroup(Results rs) throws SQLException
     public Set<String> getAllowableCategoryGroups(Container c, User u)
     {
         Set<String> types = new HashSet<>();
-        types.add("Labwork");
-        for (LabworkType type : EHRService.get().getLabworkTypes(c))
-        {
-            types.add(type.getName());
-        }
-
-        return types;
+        types.add(_labwork_category);
```

Comment:
Why is a hash set being returned if it is just being populated with one string?

Reply:
ClinicalHistoryManager.java in the EHR module expects the category types to be in a hash set. I decided to group all the labwork types into a single category for display purposes on the clinical history page to make it easier to enable/disable labwork results.